spec/factories/container_volume.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :container_volume do
+    sequence(:name) { |n| "container_volume_#{seq_padded_for_sorting(n)}" }
```

Comment:
sequence isn't really needed for this test, does not hurt much either.
(since you are overriding the name yourself)